settings.gradle

```diff
@@ -62,6 +62,7 @@ include 'java-sdk'
 
 // community modules
 include 'workflow-event-listener'
+include 'task-status-listener'
```

Comment:
task status listener does not work fully and prob. should be a separate PR later.